build.yaml

```diff
@@ -318,6 +318,7 @@ common_cli:
 
     # using the caching-layer, install all the dependencies
     pip install --upgrade pip
+    pip install -r requirements.txt ${PIP_ADDITIONAL_OPTIONS}
```

Comment:
Add an explanatory comment here. Why is it not sufficient to run the next command which parses `setup.py` which includes all of `requirements.txt`?